### ISSUE TITLE
Action_Density, write to DataSets

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#68]
I am working on this Action to write data to Dataset. Any suggestion which Dataset I should use?

(it's ok if saying `figure it out yourself`).

**Comment by drroe** [comment]
Seems like this might be OK to use DataSet_double since the X values will always be monotonic. You will just have to make sure you set the X dimension for each set correctly, e.g.
```C++
Dimension Xdim( ((double)plus_minidx + 0.5) * delta, delta, plus_maxidx - plus_minidx + 1, "Density" );
data_->SetDim(Dimension::X, Xdim);
```
You should double check my math on the minimum and # bins but you get the idea.

**Comment by drroe** [comment]
Addressed via #381
